proposals/lip-0048.md

```diff
@@ -25,32 +25,43 @@ This LIP defines the fee system in a modular way, as currently [used in the Lisk
 
 ## Rationale
 
-The fee structure presented here serves to allow defining sufficient cost to manage state growth and prevent transaction pool spamming. In principle, any transaction resulting in a state change should incur a fee. This reasoning is implemented in the `payFee` method, which can be invoked by other modules.
+### Initialization Fee
 
-### Fee Token ID
+The fee structure presented here allows for defining a sufficient cost to manage state growth. In principle, any transaction resulting in a state increase should imply a fee. This reasoning is implemented in the [`payFee`](#payfee) method, which can be invoked by other modules to charge *initialization fee* for data store initializations or some other special purposes, e.g., chain or validator registration. 
+
+For example, this method is called during the execution of the [validator registration command][lip-0057#execution] in LIP 0057: 
+```python
+Fee.payFee(VALIDATOR_REGISTRATION_FEE)
+```
+
+### Transaction Fee
+
+*Transaction fees* are introduced to motivate validators. Any remaining amount from the transaction fee, not consumed by the transaction processing, is given as a reward to the validator that includes the transaction in a block. 
+
+#### Fee Token ID
 
 Each chain can configure the token used to pay fees, as defined by `TOKEN_ID_FEE`. On the Lisk mainchain, the token used for transaction fees is the LSK token.
 
-### Minimum Fee per Transaction
+#### Minimum Fee per Transaction
 
-As introduced in [LIP 0013][lip-0013], all transactions must have a fee greater or equal to a minimum fee, which can be set to zero. For a transaction `trs`, the minimum fee is determined by its size and the proportionality constant `MIN_FEE_PER_BYTE`: 
+As introduced in [LIP 0013][lip-0013], all transactions must have a transaction fee greater or equal to a *minimum fee*, which is introduced in order to discourage validators from sending zero-cost transactions in the blocks they generate. 
```

Comment:
At this stage of the text, perhaps it is not so clear why this discourages validators. Perhaps it would be good to add a sentence explaining this. 

Example: 

As introduced in [LIP 0013][lip-0013], all transactions must have a transaction fee greater or equal to a *minimum fee*. This is always deducted from the part of the fee which is given to the block generator. This way, the minimum fee (if non zero) is designed to discourage validators from sending zero-cost transactions in the blocks they generate.

Reply:
Agreed, this makes it more clear. The nuances...